subroutines.sh

```diff
@@ -13,35 +13,110 @@ send_command() {
     log_info "tesla-control send command succeeded"
     break
   else
-	if [[ $message == *"Failed to execute command: car could not execute command"* ]]; then
-	 log_error $message
-	 log_notice "Skipping command $@ to vin $vin"
-	 break
-	else
+        if [[ $message == *"Failed to execute command: car could not execute command"* ]]; then
+         log_error $message
+         log_notice "Skipping command $@ to vin $vin"
+         break
+        else
      log_error "tesla-control send command failed exit status $EXIT_STATUS."
```

Comment:
there may be an indentation issue on this one? though should not affect the code

Reply:
I could be wrong but I bet someone might have pushed tabs w/o having the git hooks in place?  Even with the git hooks in place, no editor uses the same about of white space for a tab.

That's why we should never put tabs in code.